src/select/select.component.ts

```diff
@@ -18,6 +20,7 @@ import { Component, Input } from "@angular/core";
 })
 export class Select {
 	static selectCount = 0;
+	@Input() inlineSelect = false;
```

Comment:
I'd go with `inline = true` ... we know it's a select 😉

Reply:
Or even `type = "inline"` ... but I'm not so sure about that one

Reply:
Will we have other types of select?

Reply:
Probably not ... dropdown uses `type` for multi/single ... but even that is kinda weird (the actual `select` element uses a boolean `multiple` for multi/single)

Reply:
@cal-smith Based on our Friday discussion, would this be
```suggestion
	@Input() display: "inline";
```
or something along those lines?